I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 218, in script call
    python:
  File "game/script.rpy", line 363, in script
    $renpy.souhd.stop(channel="sound")
  File "game/script.rpy", line 363, in <module>
    $renpy.souhd.stop(channel="sound")
AttributeError: 'module' object has no attribute 'souhd'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 218, in script call
    python:
  File "game/script.rpy", line 363, in script
    $renpy.souhd.stop(channel="sound")
  File "C:\Users\Daniel\Desktop\emu\renpy\renpy-6.99.10-sdk\renpy\ast.py", line 806, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Daniel\Desktop\emu\renpy\renpy-6.99.10-sdk\renpy\python.py", line 1577, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 363, in <module>
    $renpy.souhd.stop(channel="sound")
AttributeError: 'module' object has no attribute 'souhd'

Windows-7-6.1.7600
Ren'Py 6.99.10.1227
Anechoic Chamber 0.0
